tests/load/test_glueray.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict
+
+import boto3
+import pytest
+
+
+@pytest.fixture(scope="session")
+def glue_job1(cloudformation_outputs: Dict[str, str]) -> str:
+    return cloudformation_outputs["GlueJob1Name"]
+
+
+def run_glue_job_get_status(job_name: str, arguments: Dict[str, str] = {}) -> str:
+    session = boto3.session.Session()
+    glue_client = session.client("glue")
+    job_run_id = glue_client.start_job_run(JobName=job_name, Arguments=arguments)
+
+    while True:
+        status_detail = glue_client.get_job_run(JobName=job_name, RunId=job_run_id.get("JobRunId"))
+        job_run = status_detail.get("JobRun")
+
+        status: str = job_run.get("JobRunState")
+        if "CompletedOn" in job_run:
+            return status
+
+
+@pytest.mark.skip(reason="Skipping until we make the required change to CodeBuild")
```

Comment:
Will remove this skip when we make the required change to the CodeBuild (or whichever method we use to upload the ZIP file)